lib/models/personasImport.types.ts

```diff
@@ -0,0 +1,47 @@
+import { Document, Model, Schema } from 'mongoose';
```

Comment:
Thanks for putting these types close to the code that uses them, I think that'll make them easier to maintain, I do worry that without full TS support and compilation checks these may get out of date and cause confusion, what do you think Vadim?

Reply:
Yeah, I agree with you. This is not a good way to do so because of redundant files. I though that it works only so but tried approach with `module:mongoose.XXX` and it works too. So I removed those `types.ts.` files. Please take a look on latest changes

Reply:
Ok thanks Vadim.